packages/Alert/src/alert.scss

```
@@ -1,39 +1,39 @@
-@import '~@igloo-ui/tokens/dist/base10/variables';
-@import '~@igloo-ui/tokens/dist/fonts.css';
-@import './mixins';
+@use '~@igloo-ui/tokens/dist/base10/variables';
+@use '~@igloo-ui/tokens/dist/fonts';
+@use 'mixins';
 
 :root {
   /* Default */
-  --ids-alert-font-family: #{$primary-font-family};
-  --ids-alert-font-size: #{$font-size-3};
-  --ids-alert-title-font-weight: #{$font-weight-semi-bold};
-  --ids-alert-padding: #{$space-3};
-  --ids-alert-flex-gap-component: #{$space-4};
-  --ids-alert-border-radius: #{$border-radius-sm};
-  --ids-alert-box-shadow: #{$shadow-6};
-  --ids-alert-background-color-light: #{$samoyed};
-  --ids-alert-background-color-darker: #{$grey-100};
+  --ids-alert-font-family: #{variables.$primary-font-family};
```

En changent le scope de l'import il faut changer sont utilisation dans la l'utilisation de la variable. 
```suggestion
  --ids-alert-font-family: #{tokens.$primary-font-family};
```

---

Voilà, j'ai appliqué ce changement partout